pkg/reconciler/autoscaling/hpa/controller.go

```diff
@@ -22,7 +22,7 @@ import (
 	networkingclient "knative.dev/networking/pkg/client/injection/client"
 	sksinformer "knative.dev/networking/pkg/client/injection/informers/networking/v1alpha1/serverlessservice"
 	kubeclient "knative.dev/pkg/client/injection/kube/client"
-	hpainformer "knative.dev/pkg/client/injection/kube/informers/autoscaling/v2beta1/horizontalpodautoscaler"
+	hpainformer "knative.dev/pkg/client/injection/kube/informers/autoscaling/v2beta2/horizontalpodautoscaler"
```

Comment:
WDYT about making the API change to v2beta2 separate PR wise? It seems like that'd be a good change in isolation to the rest in this PR even. Would make the actual changes a lot smaller too.

Reply:
OK, I will do that.

Reply:
@markusthoemmes  https://github.com/knative/serving/pull/12278